### PR TITLE
Bump to 1.0.6202 and fix bump script

### PR DESCRIPTION
**Pull request by Tim108**
I heard the server was getting an upgrade to this version so it would be nice if we could also use this version in development.

I added a tag to this commit, not sure where that went. Hopefully it will work when pushed to master.

### CODE REVIEW

**Review by thumbnail — APPROVED** [review]
> I added a tag to this commit, not sure where that went. Hopefully it will work when pushed to master.

There's no tag on the commit. without a commit it will not be built regardless of the branch. 

_however_ it's fine to create a release after merging this PR; which'll create a PR for you.


PR seems okay; let's await @Gluestick's approval,, as i'm not too familiar with this repo.

File: README.md [hunk 1]
```diff
@@ -22,12 +22,12 @@ To start datomic, execute the following command in the container:
 ```yml
 services:
   datomic:
-    image: quay.io/nedap/datomic:0.9.6021
+    image: quay.io/nedap/datomic:1.0.6202
```
Comment (Gluestick) [inline README.md:25]:
```suggestion
    image: quay.io/nedap/datomic:1.0.6202.0
```

File: Dockerfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ ARG DATOMIC_REPO_USER
 ARG DATOMIC_REPO_PASS
 ARG DATOMIC_LICENSE
 
-ENV DATOMIC_VERSION 0.9.6021
+ENV DATOMIC_VERSION 1.0.6202
```
Comment (Gluestick) [inline Dockerfile:7]:
```suggestion
ENV DATOMIC_VERSION 1.0.6202.0
```

File: README.md [hunk 3]
```diff
@@ -22,12 +22,12 @@ To start datomic, execute the following command in the container:
...
     command: ["./bin/transactor", "transactor.properties"]
     environment:
       DATABASES: "example,another"
     volumes:
-      - datomic-data:/opt/datomic-pro-0.9.6021/data
+      - datomic-data:/opt/datomic-pro-1.0.6202/data
```
Comment (Gluestick) [inline README.md:30]:
```suggestion
      - datomic-data:/opt/datomic-pro-1.0.6202.0/data
```

File: bump.sh [hunk 4]
```diff
@@ -7,4 +7,5 @@ if [[ $# -eq 0 ]]; then
 fi
 
 sed -i '' -e "s/^ENV DATOMIC_VERSION [0-9]\.[0-9]\.[0-9]*/ENV DATOMIC_VERSION $1/" Dockerfile
-sed -i '' -e "s/datomic-pro-starter:[0-9]\.[0-9]\.[0-9]*/datomic-pro-starter:$1/" README.md
+sed -i '' -e "s/quay.io\/nedap\/datomic:[0-9]\.[0-9]\.[0-9]*/quay.io\/nedap\/datomic:$1/" README.md
+sed -i '' -e "s/datomic-pro-[0-9]\.[0-9]\.[0-9]*/datomic-pro-$1/" README.md
```
Comment (Gluestick) [inline bump.sh:11]:
```suggestion
sed -i '' -e "s/quay.io\/nedap\/datomic:[0-9]\.[0-9]\.[0-9]*\.[0-9]*/quay.io\/nedap\/datomic:$1/" README.md
sed -i '' -e "s/datomic-pro-[0-9]\.[0-9]\.[0-9]\.[0-9]**/datomic-pro-$1/" README.md
```

**Review by Gluestick — CHANGES_REQUESTED** [review]
The version scheme for this project was changed to `a.b.c.d` instead of `a.b.c` to allow bugfix/tweak releases for the same datomic version. I somehow missed the readme/bump script when I did that...

Would you mind changing those to incorporate an extra `.\d`? You could then push the tag `1.0.6202.0` once merged to master.

File: Dockerfile (hunk 2 above)
Comment (Tim108) [reply Dockerfile]: Wait, isn't it going to try to pull datomic version 1.0.6202.0 now? Which doesn't exist, because the .0 is only for this project?

Comment (Gluestick) [reply Dockerfile]: Oops, good point, this one doesn't apply.

File: Dockerfile [hunk 5]
```diff
@@ -4,7 +4,7 @@ ARG DATOMIC_REPO_USER
 ARG DATOMIC_REPO_PASS
 ARG DATOMIC_LICENSE
 
-ENV DATOMIC_VERSION 0.9.6021
+ENV DATOMIC_VERSION 1.0.6202.0
```
Comment (Gluestick) [inline Dockerfile:7]:
```suggestion
ENV DATOMIC_VERSION 1.0.6202
```

As mentioned this one actually doesn't need the 4th digit 😅

Comment (Tim108) [reply Dockerfile:7]: Then i think we should change the script to also make the script like this, right?

Comment (Gluestick) [reply Dockerfile:7]: Ah, right. You could use `sed` to strip away the 4th digit from the version number when setting this value, so `bump.sh 1.0.6202.0` just works.

Comment (Tim108) [reply Dockerfile]: See newest commit 👌

**Review by thumbnail — APPROVED** [review]

**Pull request merged**